evm_loader/test_cli_emulate.py

```diff
@@ -139,96 +165,250 @@ def test_successful_cli_emulate(self):
+        }
+
+        emulate_result = emulate_external_call(self.ethereum_caller.hex(),
+                                               self.contract.ethereum_id.hex(),
+                                               trx_data.hex())
 
         self.compare_tmpl_and_emulate_result(tmpl_json, emulate_result)
 
         # no changes after the emulation
         self.assertEqual(self.spl_token.balance(self.token_acc1), balance1 + mint_amount - transfer_amount)
         self.assertEqual(self.spl_token.balance(self.token_acc2), balance2 + transfer_amount)
 
-    def test_unsuccessful_cli_emulate(self):
-        balance1 = self.spl_token.balance(self.token_acc1)
-        balance2 = self.spl_token.balance(self.token_acc2)
-        mint_amount = 100
-        self.spl_token.mint(self.token, self.token_acc1, mint_amount)
-        self.assertEqual(self.spl_token.balance(self.token_acc1), balance1 + mint_amount)
-        self.assertEqual(self.spl_token.balance(self.token_acc2), balance2)
-
-        transfer_amount = self.spl_token.balance(self.token_acc1) + 1
-        (trx_data, result)\
-            = self.contract.transfer_ext(self.ethereum_caller,
-                                         self.token, self.token_acc1, self.token_acc2, transfer_amount * (10 ** 9),
-                                         self.acc.public_key()._key)
-        self.assertEqual(result, None)
-
-        emulate_result = emulate_external_call(self.ethereum_caller.hex(),
-                                               self.contract.ethereum_id.hex(),
-                                               trx_data.hex())
-        tmpl = """ {
+        tmpl_json = {
+            "account_metas": [
+                {
+                    "pubkey": '' + tokenkeg,
+                    "is_signer": False,
+                    "is_writable": False,
+                },
+                {
+                    "pubkey": '' + self.token,
+                    "is_signer": False,
+                    "is_writable": False,
+                },
+                {
+                    "pubkey": '' + self.token_acc1,
+                    "is_signer": False,
+                    "is_writable": True,
+                },
+                {
+                    "pubkey": '' + self.token_acc2,
+                    "is_signer": False,
+                    "is_writable": True,
+                },
+                {
+                    "pubkey": str(self.acc.public_key()),
+                    "is_signer": True,
+                    "is_writable": False,
+                },
+            ],
             "accounts": [
                 {
-                    "account": "CONTRACT_ACCOUNT",
-                    "address": "CONTRACT_ETHEREUM_ID",
-                    "code_size": null,
-                    "contract": "CONTRACT_CODE_ACCOUNT",
-                    "new": false,
-                    "writable": true
+                    "account": '' + self.contract.contract_account,
```

Comment:
If you need string, you can try `str(self.contract.contract_account)`

Reply:
OK. Will do it in the next task